pkgs/development/python-modules/scrapinghub/default.nix

```diff
@@ -0,0 +1,30 @@
+{ buildPythonPackage, fetchPypi, lib
+, requests, retrying, six
+, glibcLocales
+}:
+
+buildPythonPackage rec {
+  pname = "scrapinghub";
+  version = "2.1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "08yhx88ihfhq4snywq9hjy9l6y0cm51cwnb69hc2f0k5qayfpqlh";
+  };
+
+  propagatedBuildInputs = [ requests retrying six ];
+
+  # Prevent unicode decoding error in setup.py
```

Comment:
Should not be needed anymore...What's the exact issue?

Reply:
To be honest, I don't see it anymore either :/

Reply:
probably you wrote these expressions while using an older Nixpkgs (18.09)

Reply:
I think I did